transform/snowflake-dbt/models/events/nightly/user_events_by_date.sql

```diff
@@ -0,0 +1,240 @@
+
+
+WITH min_active              AS (
+    SELECT
+        user_actual_id as user_id
+      , COALESCE(user_events_telemetry.user_id, IFF(LENGTH(user_events_telemetry.context_server) < 26, NULL, user_events_telemetry.context_server),
+                          IFF(LENGTH(user_events_telemetry.context_traits_userid) < 26, NULL,user_events_telemetry.context_traits_userid),
+                          IFF(LENGTH(user_events_telemetry.context_server) < 26, NULL, user_events_telemetry.context_server)) as server_id
+
+      , MAX(user_actual_role) AS user_actual_role
+      , min(received_at::date) AS min_active_date
+      , CASE WHEN MAX(received_at::date) < CURRENT_DATE
+            THEN CURRENT_DATE ELSE
+            MAX(received_at::date) END AS MAX_ACTIVE_DATE 
+    FROM "ANALYTICS".events.user_events_telemetry
+    WHERE user_actual_id is not null
+    GROUP BY 1, 2),
+
+     dates                   AS (
```

Comment:
spacing?

Reply:
working on fixing all the spaces

Reply:
Apologies. This no longer applies, as this code belonged to a different model.